examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -94,12 +113,15 @@
    "source": [
     "# Deploy Logistic Regression Model\n",
     "\n",
-    "While 74% accuracy on a CIFAR binary classification task is significantly below state of the art, it's already much better than the 50% accuracy of random guessing that you're getting right now without machine learning.\n",
+    "While 68-74% accuracy on a CIFAR binary classification task is significantly below state of the art, it's already much better than the 50% accuracy of random guessing that you're getting right now without machine learning.\n",
```

Comment:
Change "that you're getting" to "that your application is getting." Also, if you can think of a better verb than "getting" go ahead and change it (I know that was my word choice).